nominees/youth-explorer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Youth Explorer",
```

Comment:
@nathanbaleeta @lacabra - The name here is Youth Explorer, but the repository points to 'WaziMap'. I am not sure if it is the same product or not. At minimum, we should add an alias for WaziMap, or change the name of this submission,

Reply:
@sabinevidal What is the relationship between Wazimap-NG and Youth Explorer?

Reply:
Hi @nathanbaleeta and @conradsp 
From what I could see '[WaziMap](https://openup.org.za/projects/wazimap)' was the original software. '[Youth Explorer](https://openup.org.za/projects/youth-explorer)' is an iteration of it which has been built out specifically to track how the census data gathered in WaziMap relates to youth in South Africa. Repo is 'WaziMap' but website is Youth Explorer. 
Would it be better to then actually resubmit this as just WaziMap?

Reply:
@sabinevidal thank you for the quick response. On this same branch, kindly add an entry for WaziMap to allow us compare the information under question before we can proceed to merge anything.

**PS:** No need to resubmit on a new pull request, use this same pull request. Thanks

Reply:
@nathanbaleeta 
Have added an entry for WaziMap, had to create a PR for this branch https://github.com/unicef/publicgoods-candidates/pull/868 
Hope this helps!

Reply:
@conradsp Replaced Youth Explorer with WaziMap in commit 557c74